spec/controllers/events_spec.cr

```diff
@@ -36,6 +36,31 @@ describe Events do
       body.should contain(EventsHelper.mock_event(id, event_start, event_end, system_id, room_email, host, {"foo" => 123}))
     end
 
+    it "metadata extension endpoint should filter by extension data" do
+      WebMock.stub(:post, "https://login.microsoftonline.com/bb89674a-238b-4b7d-91ec-6bebad83553a/oauth2/v2.0/token")
```

Comment:
This would be a slightly better structure for the specs here
```suggestion
    describe "/extension_metadata" do
      it "filters by extension data" do
        WebMock.stub(:post, "https://login.microsoftonline.com/bb89674a-238b-4b7d-91ec-6bebad83553a/oauth2/v2.0/token")
```

Reply:
Yeah I made an issue to fix up the specs I'll do that then